src/dropdown.tsx

```diff
@@ -85,13 +89,15 @@ function Dropdown(props: DropdownProps, ref: React.Ref<DropdownRef>) {
       contentStyle={[contentStyle, menuContentStyle]}
       testID={menuTestID}
     >
-      <CustomMenuHeader
-        label={label}
-        toggleMenu={toggleMenu}
-        resetMenu={resetMenu}
-        value={value}
-        multiSelect={false}
-      />
+      {!props.hideMenuHeader && (
```

Comment:
Can you get the value from the props above where we are de structuring it and assign it value of false.

Reply:
Ready!